Add Bounty Context to Treasury Page (#5307)
w3f/polkadot-wiki: docs/learn/learn-guides-bounties.md

[Bounties page](https://polkadot.js.org/apps/#/bounties). You can then proceed with assigning
curators to the bounty.

To minimize storage on chain in the same way as any proposal, bounties don't contain contextual

> information. When a user submits a bounty spending proposal, they will need to find an off-chain way

Reviewer: ```suggestion
information. When a user submits a bounty spending proposal, they will need to find an off-chain medium
```

> to explain the proposal through for example [Polkassembly](https://polkadot.polkassembly.io/opengov)

Reviewer: ```suggestion
to explain the proposal, for example a bounty proposal document on Polkassembly [Polkassembly](https://polkadot.polkassembly.io/opengov)
```

or [Subsquare](https://polkadot.subsquare.io/).
[This template](https://docs.google.com/document/d/1-IBz_owspV5OcvezWXpksWDQReWowschD0TFuaVKKcU/edit?usp=sharing)

> can help as a checklist of all needed information for OpenGov voters to make an informed decision.

Reviewer: ```suggestion
can be used to submit all the information needed by OpenGov voters to make an informed decision.
```
